src/SprykerEco/Zed/Payone/Business/TransactionStatus/TransactionStatusUpdateManager.php

```diff
@@ -125,21 +128,21 @@ protected function transformCurrency(TransactionStatusUpdateInterface $request):
         $balance = $request->getBalance();
         $balanceAmountInCents = round((float)$balance * 100);
 
-        if (method_exists($request, 'setBalance')) {
+        if ($request instanceof TransactionStatusRequest) {
```

Comment:
Maybe it's better to create some variable for this check instead of calling `instance of` tree times?